templates/components/card.html

```diff
@@ -26,13 +38,19 @@
       {% endif %}
     {% else %}
       <h{{ headingLevel }} class="tna-card__heading tna-heading tna-heading--{{ headingSize|default:'m' }}">
+      <h{{ headingLevel }} class="tna-card__heading tna-heading tna-heading--{{ headingSize|default:'m' }} {% if highlight %}tna-card__heading--has-highlight{% endif %}" >
```

Comment:
Is this duplicated opening heading tags?

Reply:
oops, good spot - was when I was porting in the tna-frontend styles and making sure everything was in line with the card as it stands